realm/realm-library/src/objectServer/java/io/realm/RealmFunctions.java

```diff
@@ -15,5 +15,18 @@
  */
 package io.realm;
 
-class RealmFunctions {
+import org.bson.BsonValue;
+
+import io.realm.internal.jni.JniBsonProtocol;
+
+public class RealmFunctions {
```

Comment:
Add FIXME for the Javadoc

Reply:
Same a for the test. This was only included to have some place for doing a round trip test to JNI. This will be superseeded by #6810.